tests/portfolio/test_ledger_entries.py

```diff
@@ -2,19 +2,25 @@
 import pytest
 from playwright.sync_api import Page, expect
 
+from actions.utils import wait_for_toast_confirmation
+
 @pytest.mark.usefixtures("page", "auth", "risk_accepted", "continuous_market")
 def test_ledger_entries_downloads(page: Page):
     page.goto("/#/portfolio")
     page.get_by_test_id("Ledger entries").click()
     expect(page.get_by_test_id("ledger-download-button")).to_be_enabled()
-
     # 7007-LEEN-001
+    page.get_by_test_id("ledger-download-button").click()
+    #7007-LEEN-009
+    expect(page.get_by_test_id("toast-content")).to_contain_text(("Your file is ready"))
```

Comment:
Do we need a wait at all here? Or increase timeout etc?

Reply:
The current wait build in an expect is sufficient